frontend/src/components/RouteTab/index.tsx

```diff
@@ -47,6 +48,7 @@ interface RouteTabProps {
 	}[];
 	activeKey: TabsProps['activeKey'];
 	onChangeHandler?: VoidFunction;
+	history: History<unknown>;
```

Comment:
Does adding history here means user can go back to previous screen?

Reply:
added the history because in the test case it was difficult to check when the user click on tab new route is added ... so just to check that behaviour added the history object here